Packs/ServiceNow/ReleaseNotes/2_6_11.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### ServiceNow v2
+
+- Fixed an issue where in some cases a new access token could not been created.
```

Comment:
Maybe something like - Fixed an issue where in some cases the access token could not be retrieved?